azure/pom.xml

```diff
@@ -110,6 +110,31 @@
       <version>1.0.0-beta5-SNAPSHOT</version>
     </dependency>
     <dependency>
+      <groupId>com.microsoft.azure</groupId>
```

Comment:
This is not correct. `azure` is the roll-up management client and shouldn't depend on data plane packages. I'll remove in a separate PR.